seabird.proto

```diff
@@ -27,6 +27,29 @@ message Event {
     PerformActionEvent perform_action = 9;
     PerformPrivateActionEvent perform_private_action = 10;
   }
+
+  // Tags are a set of key-value items which may expose additional information
+  // about the given Event.
+  //
+  // All tags SHOULD be prefixed with something unique to that plugin or
+  // backend. All tags starting with `seabird/` are reserved for use by
```

Comment:
That seems pretty broad to me. Can we instead reserve something like `core/` or `seabird_core/`?

Reply:
Sure, that seems reasonable

Reply:
Updated to seabird-core/